tests/modules/rigidbody/test_rigidbody.py

```diff
@@ -0,0 +1,98 @@
+"""Test rigidbody module."""
+import json
+from pathlib import Path
+
+from haddock.modules.sampling.rigidbody import HaddockModule as RigidBody
+
+
+RECIPE_PATH = Path(
```

Comment:
you can import these directly
`from haddock.modules.sampling.rigidbody import DEFAULT_CONFIG`